index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>tribute page</title>
+    <link rel="stylesheet" href="style.css">
+    <style>
+        #image {
+          max-width: 100%;
+          height: auto;
+        }
+
+        img {
+            display: block;
+        }
+        
+    </style>
```

Comment:
- You have a separate file for adding your style codes so remove and separate the style codes from your HTML file.